portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['My last name is Preda!', 'I\'m from Romania!', 'I like programming！', 'My hobby is traveling!', 'When I was a kid, I wanted to be a truck driver!'];
```

Comment:
Is the space after "programming!" required?

Reply:
Well, there is no actual space after programming (try to select it), I don't know why it looks like this.

Reply:
It looks like a non-ASCII exclamation mark. Hence it is shown like this. Please delete it and replace with an ASCII one.